Bottom: a36b0c79deb8c781e0096021c2108858b5e1d7ca
Top:    7e01124fb11368643d3e244daa1aab15326f1b64
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 13:47:35 +0000

Refresh of attempt-at-producing-inter

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index feb9be5..974af9d 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1691,7 +1691,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 
   min_copper_group = MIN (bottom_group, top_group);
   max_copper_group = MAX (bottom_group, top_group);
-//  num_copper_groups = max_copper_group - min_copper_group;
 
   group_m_polyarea = calloc (max_copper_group + 1, sizeof (POLYAREA *));
